tdx-attestation-app/suse_install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# sudo rpm --import https://packman.inode.at/keys/packman.key
+sudo zypper ar -cfp 90 https://ftp.fau.de/packman/suse/openSUSE_Tumbleweed/ packman
+sudo zypper addrepo -f https://download.opensuse.org/distribution/leap/15.5/repo/oss/ openSUSE-0ss
+
+# Update
+sudo zypper --gpg-auto-import-keys refresh
+
+# Install required packages
+sudo zypper install -y -t pattern devel_basis
+sudo zypper install -y cmake
+sudo zypper install -y nlohmann_json-devel
+sudo zypper install -y libcurl4
+sudo zypper install -y libcurl-devel
+sudo zypper install -y libboost_headers1_75_0-devel
+
+# Install GuestAttestation Debian Package
+sudo rpm --nosignature -i package/azguestattestation1-1.0.5-2.x86_64.rpm
```

Comment:
Please don't submit rpm files; they are large and can get quickly outdated.

You can do rpm -ihv https://package.microsoft.com/...../azguestattestation1-1.0.5-2.x86_64.rpm

Reply:
This is only for private preview. Those packages are official release, is that correct?

Reply:
Additionally, this change is not going to main branch. Before this goes to main branch, we will release package as you mentioned above!